src/main/webapp/script.js

```diff
@@ -1,13 +1,333 @@
-/**
- * Adds a random location to the page.
- */
-function addRandomLocation() {  // eslint-disable-line no-unused-vars
-  const locations = ['Canada', 'Mexico', 'US'];
-
-  // Pick a random location.
-  const location = locations[Math.floor(Math.random() * locations.length)];
-
-  // Add it to the page.
-  const locationContainer = document.getElementById('location-container');
-  locationContainer.innerText = location;
+/** *************************
+Autocomplete Searchbar
+***************************/
+function autocomplete(inp, arr) {
+  // Create variable for the item on list currently focused on
+  let currentFocus;
+
+  // Function for when the user types into the searchbar
+  inp.addEventListener('input', function() {
+    let a; let b; let i; const val = this.value;
+
+    // Close any open autocomplete lists
+    closeAllLists();
+    if (!val) {
+      return false;
+    }
+
+    currentFocus = -1;
+
+    // Create a DIV element that will contain the items (values)
+    a = document.createElement('DIV');
+    a.setAttribute('id', this.id + 'autocomplete-list');
+    a.setAttribute('class', 'autocomplete-items');
+
+    // Append the DIV element as a child of the autocomplete container
+    this.parentNode.appendChild(a);
```

Comment:
I suggest simplifying this to reuse a div container purposed for autocompletion:

```suggestion
    let val = this.value;
    
    // Get and clear the autocomplete list
    let a = document.getElementById('autocomplete-list');
    a.innerHTML = "";
    
    // Provide no suggestions for empty input
    if (!this.value) {
      return false;
    }

    currentFocus = -1;
```

Reply:
Thank you for the suggestion! After making this change, somehow I was unable to use the arrow keys to sift through the options in the list of suggested countries under the search bar.

Reply:
That is peculiar

Reply:
Ah, I figured out why that was. It's because [this line](https://github.com/gferioli0418/travelbud/blob/4b6a5865318e4f9666c40bb1ca674af2068b1622/src/main/webapp/script.js#L49) needs to also be changed to `document.getElementById('autocomplete-list')`